MAUI/Rating/Migration.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfRating | Syncfusion 
+description: Learn all about Migrating from Syncfusion Xamarin Rating to Syncfusion .NET MAUI Rating control and more here.
+platform: maui
+control: SfRating
+documentation: ug
+---  
+
+# Migrating from Xamarin SfRating to .NET MAUI SfRating 
+
+To migrate easier from Xamarin SfRating to [.NET MAUI SfRating], we kept most of the APIs from Xamarin SfRating in MAUI SfRating. However, to maintain the consistency of API naming in MAUI SfRating, we renamed some of the APIs. The APIs changed in MAUI SfRating from Xamarin SfRating are detailed as follows:
```

Comment:
To migrate easier from Xamarin SfRating to [.NET MAUI SfRating], most of the APIs were kept from Xamarin SfRating in MAUI SfRating. However, to maintain the consistency of API naming in MAUI SfRating, some of the APIs were renamed. The APIs changed in MAUI SfRating from Xamarin SfRating are detailed as follows:

Reply:
Changed